### PR TITLE
Rest of the space management modules

### PR DESCRIPTION
**Pull request by 1egoman**
- [x] Danger Zone module
- [x] Operating hours (v1 version) module
- ~[ ] Tags module (on the rooms page)~ not being done for this release

In addition, I extracted the map out of the address module and added a boolean `readonly` prop to it that defaults to `false`. The map in the side pane for buildings will have this prop set to `true`. With this change, I think at this point the map has all the features required for this first release at least.

### CODE REVIEW

File: src/components/admin-locations-detail-modules/index.tsx [hunk 1]
```diff
@@ -659,170 +663,104 @@ export function AdminLocationsDetailModulesMetadata({space, formState, onChangeF
...
-          element: MARKER_ELEMENT,
-          anchor: 'center',
-          draggable: true,
-        });
-        this.initialMarker.setLngLat(coordinates);
-        this.initialMarker.addTo(this.map);
-        this.initialMarker.on('dragend', this.onMapPointDragged);
-        this.map.setCenter(coordinates);
-      }
+// NOTE TO SELF AND TO ANY REVIEWERS: This component needs to be extracted out into its own file
```
Comment (guscost) [inline src/components/admin-locations-detail-modules/index.tsx]: ???

Comment (1egoman) [reply src/components/admin-locations-detail-modules/index.tsx]: No longer relevant, but thanks for catching this!

File: src/components/admin-locations-building-detail/index.tsx [hunk 2]
```diff
@@ -74,7 +78,22 @@ export default function AdminLocationsBuildingDetail({ spaces, selectedSpace })
...
+            <Button onClick={() => {
+              window.location.href = `#/admin/locations/${selectedSpace.id}/edit`;
+            }}>Edit</Button>
+          </AppBarSection>
+        </AppBar>
+        {selectedSpace.latitude !== null && selectedSpace.longitude !== null ? (
+          <AdminLocationsSpaceMap
+            readonly={true}
+            space={selectedSpace}
+            address="WeWork Gramercy, 120 E 23rd St, New York City, New York 10010, United States of America"
```
Comment (guscost) [inline src/components/admin-locations-building-detail/index.tsx]: Could we mark this with HARDCODED or something?

File: src/components/admin-locations-detail-modules/index.tsx [hunk 3]
```diff
@@ -659,170 +663,102 @@ export function AdminLocationsDetailModulesMetadata({space, formState, onChangeF
...
-      }
+export function AdminLocationsDetailModulesAddress({
+  space,
+  address,
+  coordinates,
+  onChangeAddress,
+  onChangeCoordinates,
+}) {
+  return (
+    <AdminLocationsDetailModule title="Address">
```
Comment (guscost) [inline src/components/admin-locations-detail-modules/index.tsx]: Yay it's not called "card"

File: src/components/admin-locations-space-map/index.tsx [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+export default class AdminLocationsSpaceMap extends Component<any, any> {
+  map?: MapboxGL.Map;
+  geocoderInstance?: any;
+  initialMarker?: any;
+  geocoder = React.createRef<HTMLDivElement>();
+  container = React.createRef<HTMLDivElement>();
+
+  static defaultProps = { readonly: false };
+
+  componentDidMount() {
```
Comment (guscost) [inline src/components/admin-locations-space-map/index.tsx]: Bleh, use hooks!

File: src/components/admin-locations-building-detail/index.tsx (hunk 2 above)
Comment (1egoman) [reply src/components/admin-locations-building-detail/index.tsx]: Yep, will do!

**Pull request merged**